src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/Utilities/Utilities.cs

```diff
@@ -153,7 +155,7 @@ public static string GetDefaultPath()
                 throw new Exception("Invalid ARM ID");
             }
 
-            for (var i = 1; i < armFields.Length; i = i + 2)
+            for (var i = 0; i < armFields.Length; i = i + 2)
```

Comment:
@viverm shouldn't this _always_ fail with an `IndexOutOfRangeException`?

For example, if `armFields` has five elements, this loop will get the following elements:

1st (idx 0) and 2nd (idx 1)
3rd (idx 2